.github/workflows/coderabbit-resolve-check.yml

```
@@ -0,0 +1,32 @@
+jobs:
```

actionlint (1.7.7)

"on" section is missing in workflow

(syntax-check)